### PR TITLE
Session Timeouts

### PR DESCRIPTION
**Pull request by lukebussey**
Changing implementation to accept a session timeout in minutes rather than setting a session cookie. This matches the default behavior of Google Analytics in which session timeout after 30 minutes of inactivity.

### CODE REVIEW

File: dist/last-campaign.js [hunk 1]
```diff
@@ -157,6 +183,24 @@ function getCookie(name) {
...
+ * @param {string} name - name of cookie
+ * @param {date} expires - expiration date/time of cookie
+ * @param {object} options - object containing cookie options
+ * @private
+ */
+function updateExpiration(name, expires, options) {
+    var existingValue = getCookie(name);
+
+    if (existingValue) {
+        setCookie(name, existingValue, merge(options, {
```
Comment (mattduggan) [inline dist/last-campaign.js:205]: I'd recommend leaving a comment about why there is a specific method for updating the `expires` through assignment.

**Comment by mattduggan** [comment]
One comment, but otherwise g2g

File: dist/last-campaign.js (hunk 1 above)
Comment (lukebussey) [inline dist/last-campaign.js:205]: all set.

**Comment by mattduggan** [comment]
g2g

**Pull request merged**